client/src/components/CompanyRegister.js

```diff
@@ -0,0 +1,194 @@
+import React, { Component } from "react";
+import {
+  Form,
+  Input,
+  Dropdown,
+  Button,
+  TextArea,
+  Message,
+  Grid,
+  Header
+} from "semantic-ui-react";
+
+import { companyRegister } from "../api/companies";
+import { industryList } from "../utils/constants";
+
+class CompanyRegister extends Component {
+  state = {
+    companyProfile: {
+      name: "",
+      description: "",
+      industry: [],
+      email: "",
+      password: "",
+      user_id: ""
+    },
+    error: null,
+    success: null
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    companyRegister(this.state.companyProfile)
+      .then(data => {
+        console.log(data);
+        if (data.success) {
+          this.setState({
+            companyProfile: {
+              name: "",
+              description: "",
+              industry: [],
+              email: "",
+              password: ""
+            },
+            success: true
+          });
+        } else {
+          this.setState({});
+        }
+        this.resetForm(data.success);
+      })
+      .catch(err => {
+        this.setState({ success: false, error: err });
+      });
+  };
+
+  handleIndustry = (event, data) => {
+    const selectedIndustries = data.value;
```

Comment:
just for you to know, you could use destructuring to write this line as such `const { value: selectedIndustries} = data` ... this tells to `destructure` _value_ from _data_ object then rename it to _selectedIndustries_  ... you can read more about this here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment


you don't need to implement it like this .. but it's something in ES6 and it's good to learn